### PR TITLE
Update cri-tools to v1.27.1

### PR DESCRIPTION
**Pull request by git-jxj**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Update cri-tools from v1.27.0 to v1.27.1 in the dependency manifest and the Linux/Windows node setup scripts, including the corresponding download checksums.

#### Which issue(s) this PR fixes:

NONE

#### Special notes for your reviewer:

NONE

#### Does this PR introduce a user-facing change?

```release-note
Updated cri-tools to v1.27.1.
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
@git-jxj I had to update the release bucket to make the artifacts available, now they're there: https://console.cloud.google.com/storage/browser/k8s-artifacts-cri-tools/release/v1.27.1

**Comment by saschagrunert** [comment]
@git-jxj can you please sign-off the CLA?

**Comment by git-jxj** [comment]
> @git-jxj can you please sign-off the CLA?

Done, thanks. PTAL

File: cluster/gce/gci/configure.sh [hunk 1]
```diff
@@ -369,7 +369,7 @@ EOF
   fi
 
   echo "Downloading crictl"
-  local -r crictl_path="https://storage.googleapis.com/k8s-artifacts-cri-tools/release/${crictl_version}"
+  local -r crictl_path="https://console.cloud.google.com/storage/browser/k8s-artifacts-cri-tools/release/${crictl_version}/crictl-${crictl_version}-linux-amd64.tar.gz"
```
Comment (saschagrunert) [inline cluster/gce/gci/configure.sh:372]: We should leave that as-is.

**Review by saschagrunert — CHANGES_REQUESTED** [review]

File: cluster/gce/gci/configure.sh (hunk 1 above)
Comment (git-jxj) [reply cluster/gce/gci/configure.sh]: fixed

**Review by saschagrunert — APPROVED** [review]
Thanks! LGTM when CI is happy as well.

**Comment by saschagrunert** [comment]
Closing in favor of https://github.com/kubernetes/kubernetes/pull/119933

**Pull request closed without merging**